FunctionHomework/src/CalculateMax/CalculateMax.java

```diff
@@ -0,0 +1,29 @@
+package CalculateMax;
+
+import java.util.Scanner;
+
+/**
+ * @author Codrin
+ *
+ */
+public class CalculateMax {
+    public static void main(String[] args) {
+        System.out.println("Enter the first number: ");
+        Scanner keyboard = new Scanner(System.in);
+        int a = keyboard.nextInt();
+        System.out.println("Enter the second number: ");
+        int b = keyboard.nextInt();
+        System.out.println("Enter the third number: ");
+        int c = keyboard.nextInt();
+        System.out.println("Your biggest number is: " + getMax(getMax(a , b), c));
+    }
+    private static int getMax(int n1, int n2) {
```

Comment:
it works, a simpler way woud be:
if(n1<n2)
  return n2;
else
  return n1;

Also, javadoc!